/**:
  odom_node:
    ros__parameters:
      odom_frame: odom
      base_frame: base_link
      odometry_topic: odometry/wheel_encoders
      odometry_queue_size: 5 # (int)
      encoder_queue_size: 1 # (int)
      encoder_topic: wheel_encoders
      # do_broadcast_transform: True  # this is set in the launch file

      # Amount of seconds to use previous updates of the wheel encoders to calculate the velocity and position(encoders update freq = 50 Hz)
      max_history_time: 0.08  # [s]

      wheel_radius: 0.0329 # [m] (double)
      ticks_per_rev: 3840. # (double)
      wheel_base: 0.1405 # [m] (double)

      pose_variances: [0.01, 0.01, 0., 0., 0., 0.01] # position [x, y, z] orientation [roll, pitch, yaw]
      # only linear.x and angular.z is used
      twist_variances: [3.0e-6, 0.0, 0.0, 0.0, 0.0, 0.04] # linear [x', y', z'] angular [roll', pitch', yaw']
